appveyor.yml

```diff
@@ -42,6 +43,23 @@ test: off
 deploy: off
 
 for:
+-
+  matrix:
+    only:
+      - APPVEYOR_BUILD_WORKER_IMAGE: Ubuntu
+
+  clone_folder: ~/projects/ros_workdir/reactos-cov
```

Comment:
"-cov"?

Reply:
I too have no idea what it means: simply copied it from initial @AmineKhaldi's Windows case.